src/sexptypes.cpp

```diff
@@ -129,10 +124,14 @@ void get_full_type_inner(SEXP sexp, SEXP rho, full_sexp_type &result,
                 return;
```

Comment:
Can we end at active bindings? We know that an active binding means the symbol is bound to a function that will be called automatically when the symbol is looked up or reassigned. So, adding the type of function after active binding may not be insightful. Also, ending at active binding lets our script show the type of this symbol as ACTIVE_BINDING which is a useful thing to know in itself.